### PR TITLE
OCM-7263 | chore: Added additional aliases to the `rosa list kubeletconfig` command

### PR DESCRIPTION
**Pull request by robpblake**

### CODE REVIEW

File: cmd/list/kubeletconfig/cmd_test.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ var _ = Describe("List KubeletConfig Command", func() {
 			Expect(cmd.Use).To(Equal(use))
 			Expect(cmd.Short).To(Equal(short))
 			Expect(cmd.Long).To(Equal(long))
-			Expect(cmd.Aliases).To(ContainElements(alias))
+			Expect(cmd.Aliases).To(ContainElements("kubeletconfig", "kubelet-config", "kubelet-configs"))
```
Comment (marcolan018) [inline cmd/list/kubeletconfig/cmd_test.go:32]: can we use ContainElements(aliases) here?

Comment (robpblake) [reply cmd/list/kubeletconfig/cmd_test.go]: @marcolan018 Good point. I've made that change.

**Pull request merged**